src/Uno.UI.Runtime.Skia.X11/X11XamlRootHost.cs

```diff
@@ -99,22 +99,32 @@ public X11XamlRootHost(X11WindowWrapper wrapper, Window winUIWindow, XamlRoot xa
 		_visibilityCallback = visibilityCallback;
 		_configureCallback = configureCallback;
 
-		_renderTimer = new DispatcherTimer();
-		_renderTimer.Interval = TimeSpan.FromSeconds(1.0 / X11ApplicationHost.RenderFrameRate); // we're on the UI thread
-		_renderTimer.Tick += (sender, o) =>
+		// running XConfigureEvent and rendering callbacks on a timer is necessary to avoid freezing in certain situations
+		// where we get spammed with these events. Most notably, when resizing a window by dragging an edge, we'll get spammed
+		// with XConfigureEvents.
+		_timer = new Timer(static obj =>
```

Comment:
We can't directly pause the `System.Threading.Timer` but I think there are ways to get around this, if we're careful enough. However, I don't think it matters. The `Timer` actually sleeps (i.e. the equivalent of a Thread.Sleep) so waking up and immediately sleeping again 60 times a second should be more than fine.

Reply:
The issue here is about CPU idle time, which may prevent CPU to go in low-power modes. It's generally best to avoid doing work we know does not need to be done.